addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -57,15 +57,23 @@
                                 <br/>
                             </div>
 
-                            <t t-if="o.partner_bank_id._is_qr_iban()">
+                            <t t-if="o.partner_bank_id._is_qr_iban() or o.partner_bank_id._is_iso11649_reference()">
                                 <div class="swissqr_text title">
-                                    <span>Reference</span>
+                                    <span class="title">Reference</span>
                                 </div>
+                            </t>
+                            <t t-if="o.partner_bank_id._is_qr_iban()">
                                 <div class="swissqr_text content">
                                     <span t-esc="o.space_qrr_reference(o.payment_reference)"/><br/>
                                     <br/>
                                 </div>
                             </t>
+                            <t t-if="o.partner_bank_id._is_iso11649_reference()">
```

Comment:
Instead of calling _is_iso11649_reference() and _is_qr_iban, I suggest using 2 t-set to store the results of these calls and reuse them as variables.

Reply:
changed it